music/migrations/0002_song_is_favorite.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.0.7 on 2018-07-07 16:03
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('music', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='song',
+            name='is_favorite',
+            field=models.BooleanField(default=False),

```

Comment:
__pycache__ folder should not be in git, add it to gitignore and remove it from git